apps/governance/src/routes/proposals/Proposals.graphql

```diff
@@ -383,6 +383,122 @@ fragment IUpdateNetworkParameterFields on UpdateNetworkParameter {
   }
 }
 
+fragment INewSpotMarket on NewSpotMarket {
+  instrument {
+    name
+    code
+    product {
+      ... on SpotProduct {
+        name
```

Comment:
This likely do not exist anymore in the last preview. could you test that code again staging please?